src/core/connectors.js

```diff
@@ -2237,6 +2237,13 @@ const connectors = [{
 	],
 	js: 'connectors/xpn.js',
 	id: 'xpn',
+}, {
+	label: 'GotRadio',
+	matches: [
+		'*://player.gotradio.com/*',
+	],
+	js: 'connectors/gotradio.js',
+	id: 'GotRadio',
```

Comment:
```suggestion
	id: 'gotradio',
```

Connector ID's should be lowercase